compose/um-metrics-02/docker-compose.yml

```diff
@@ -0,0 +1,110 @@
+version: "3.9"
+
+volumes:
+  prometheus-data: {}
+  grafana-data: {}
+  um_data: {}
+  um_logs: {}
+  um_licence: {}
+  um_users: {}
+
+networks:
+  n1:
+    external: false
+
+services:
+  umnode:
+    image: ${UM_CONTAINER_IMAGE_NAME}
+    ports:                         
+     - 9000:9000
+     - 9200:9200
+    volumes:
+     - um_data:/opt/softwareag/UniversalMessaging/server/umserver/data
+     - um_logs:/opt/softwareag/UniversalMessaging/server/umserver/logs
+     - ${UM_LICENCE_LOCATION}:/opt/softwareag/UniversalMessaging/server/umserver/licence
```

Comment:
```suggestion
     - ${UM_LICENCE_LOCATION}:/opt/softwareag/UniversalMessaging/server/umserver/licence/
```

Reply:
Add a slash to quickly tell the yaml reader this is a directory mount, not a file mount.
Besides, add a note in the README that the directory needs to contain a file called exactly "licence.xml"